Applications/Browser.md

```diff
@@ -242,11 +242,50 @@ Alternatively, you can select only certain months in a year using the filter by
 ![Timelapse](_images/Timelapse.png)
 Fig 16: Browser timelapse tool with settings highlighted
 
+### Statistical Ananlysis
 
+There are quite a lot of statistical analyses that can be done in the Browser itself. Depending on the type of data you are looking at, you can look at the distribution of the pixel values, view the spectral signature of the region using the **Spectral Explorer**(![spectral](_images/icons/Spectral_Explorer.png)) or, in case of indices, look at the change in value over time using the **Statistical Info** (![timeseries](_images/icons/HistIcon.png)) feature. 
```

Comment:
```suggestion
There are quite a lot of statistical analyses that can be done in the Browser itself. Depending on the type of data you are looking at, you can look at the distribution of the pixel values, view the spectral signature of the region using the **Spectral Explorer**(![spectral](_images/icons/Spectral_Explorer.png)) or, in case of indices, look at its change over time using the **Statistical Info** (![timeseries](_images/icons/HistIcon.png)) feature. 
```